### ISSUE TITLE
Multiple monitor support

### ISSUE THREAD

**Issue by joelvaneenwyk** [x13pixels/remedybg-issues#90]
Would love to be able to have some windows (e.g. output) take up an entire separate screen/window. As it stands now I don't see any way to easily have RemedyBG span multiple monitors.

**Comment by BigDru** [comment]
I'm glad I checked the existing issues before submitting a new request. 

I would also like to undock panels and have them float on separate monitors. I have 4 monitors and having everything be forced to one monitor is unfortunate. 

I don't know the code base so I can't specifically comment on implementation, but I saw that the docking branch of imgui is a dependency so this feature should be in the native ui framework. Perhaps it's as simple as enabling via the imgui configuration. 

Eg (lines 51 & 52): 
https://github.com/ocornut/imgui/blob/9937660b1cdf79a98b74647608f0e57a62a36e62/examples/example_glfw_opengl2/main.cpp#L51

```
    io.ConfigFlags |= ImGuiConfigFlags_DockingEnable;         // Enable Docking
    io.ConfigFlags |= ImGuiConfigFlags_ViewportsEnable;       // Enable Multi-Viewport / Platform Windows
```
